src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Menus.vsct

```diff
@@ -251,6 +222,14 @@
       <IDSymbol name="cmdidAddNewItemBelow" value="0x2004" />
       <IDSymbol name="cmdidAddExistingItemBelow" value="0x2005" />
     </GuidSymbol>
+
+    <!-- The guids and ids below are the same as F#'s legacy project values.
+         We do this to allow hotkey alt+up/down to work in legacy and CPS projects.
+         When F# fully switches over to CPS for older projects, we can get rid of this and use guidManagedProjectSystemOrderCommandSet instead. -->
```

Comment:
I don't understand this comment?  `guidManagedProjectSystemOrderCommandSet` is what you are defining.

Reply:
Nevermind I see now.